src/flags.rs

```diff
@@ -19,19 +20,21 @@ pub struct DenoFlags {
   pub reload: bool,
   pub allow_write: bool,
   pub allow_net: bool,
+  pub eval: Vec<String>,
 }
 
 pub fn print_usage() {
   println!(
     "Usage: deno script.ts
 
---allow-write      Allow file system write access.
---allow-net        Allow network access.
--v or --version    Print the version.
--r or --reload     Reload cached remote resources.
--D or --log-debug  Log debug output.
---help             Print this message.
---v8-options       Print V8 command line options."
+--allow-write          Allow file system write access.
+--allow-net            Allow network access.
+-v or --version        Print the version.
+-e or --eval [SCRIPT]  Evaluate a script from the command line, may be passed multiple times.
```

Comment:
Do other languages support multiple eval scripts here? Neither python or node do.

Reply:
Perl does, it's not a big deal or anything, but personally, I like writing separate lines as `-e'...'` better than just separating them with a `;` in a single `eval` flag. Particularly in shell scripts, it can be kind of nice if you want to avoid a Here-Document. For example:

```bash
deno -e "console.log('$HELLO')" \
     -e "console.log('$WORLD')"
```

You could do something like this if you wanted to use env vars in a tiny script for some reason.

Reply:
Seems easy to support - I'm for it.